tests/conftest.py

```diff
@@ -97,6 +98,25 @@ def bake_examples_compiled_dev_version(cookies):
     return _handle_cookiecutter_errors(result)
 
 
+@pytest.fixture
+def bake_examples_url_extensions(cookies):
+    """
+    setting url parameters
+    """
+    result = cookies.bake(extra_context={
+        "author_name": "test",
+        "include_example_code": "y",
+        "project_url": "https://sunpy.org",
+        "github_repo": "sunpy/sunpy",
+        "sourcecode_url": "",
```

Comment:
Does this automatically take the default value? (My assumption would be no)

Reply:
ah,  yep I'd forgotten about that, i'll write a test for it